NuGet.config

```diff
@@ -8,7 +8,6 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from DotNet-msbuild-Trusted -->
-    <add key="darc-pub-DotNet-msbuild-Trusted-7a09d5e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-DotNet-msbuild-Trusted-7a09d5ee/nuget/v3/index.json" />
```

Comment:
Why is that removed?

Reply:
That got removed by the Maestro backport sync client. I asked @premun about it. We aren't yet sure why it gets removed.

Reply:
Resolving this conversation so that we can merge the PR.